lib/app-capitano.coffee

```diff
@@ -20,7 +20,7 @@ actions = require('./actions')
 events = require('./events')
 
 capitano.permission 'user', (done) ->
-	require('./utils/patterns').exitIfNotLoggedIn()
+	require('./utils/patterns').checkLoggedIn()
```

Comment:
If you haven't, could you test that this change in particular works? It depends on Capitano correctly handling a rejected promise here. (Previously, `process.exit()` was called so it didn't matter...)

Reply:
Checked.